docs/learn/learn-staking.md

```diff
@@ -9,189 +9,357 @@ slug: ../learn-staking
+
+## Nominated Proof-of-Stake Overview
+
+{{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} implements [Nominated Proof-of-Stake (NPoS)](learn-consensus.md/#nominated-proof-of-stake),
+a relatively novel and sophisticated mechanism to select the validators who are allowed to participate in 
+its [consensus](learn-consensus.md) protocol. The NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} 
+holders to participate as nominators. NPoS encourages {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }}
+holders to participate as `nominators`. 
+
 Nominators may back up to
 {{ polkadot: <RPC network="polkadot" path="consts.staking.maxNominations" defaultValue={16}/> :polkadot }}
 {{ kusama: <RPC network="kusama" path="consts.staking.maxNominations" defaultValue={24}/> :kusama }}
-validators as trusted validator candidates. Both validators and nominators lock their tokens as
-collateral and receive staking rewards.
+validators as trusted validator candidates, and the network will automatically distribute the stake among validators in
+an even manner. Also, in NPoS the stake of both nominators and validators can be slashed.  For an in-depth review of NPoS see
+[this](https://research.web3.foundation/en/latest/polkadot/NPoS/index.html) research article.
+
+### Nominating Validators
+
+The action of nominating consists of a) locking or
+bonding tokens (stake) on the chain, and b) nominating a set of validator candidates to whom the stake will
+be allocated. {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} uses tools ranging
+from election theory to game theory to discrete optimization, to develop an efficient validator
+selection process that offers fair representation and security, thus avoiding uneven power and
+influence among validators. The election algorithms used by {{ polkadot: Polkadot :polkadot }}{{ kusama: Kusama :kusama }} are based on the Proportional Justified Representation (PJR) methods like [Phragmen](learn-phragmen.md). For more information about PJR methods visit [this](https://research.web3.foundation/en/latest/polkadot/NPoS/1.%20Overview.html?highlight=proportional%20justified%20representation#) research article.
+
+### Eras and Sessions
+
+The stake from nominators is used to increase the number of tokens held by such candidates, increasing their chance of being selected by the election algorithm for block
+production during a specific `era`. An era is a period of {{ polkadot: 24 hours :polkadot }}{{ kusama: 6 hours :kusama }} during
+which an `active set` of validators is producing blocks and performing other actions on chain. This
+means that not all validators are in the active set and such set changes between eras. Each era is
+divided into 6 epochs or `sessions` during which validators are assigned as block producers to
```

Comment:
@kianenigma I introduced the terms here, and defined them appropriately because they are used later in the page and I believe is important for the reader to know what these things are in general, and they must know it as soon as possible before the explanations become complex. Let me know if you agree with this.